.scripts/run_win_build.bat

```diff
@@ -51,6 +51,10 @@ if NOT [%flow_run_id%] == [] (
 
 call :end_group
 
+call :start_group "Enable git symlinks on Windows"
+git config --global core.symlinks true
+call :end_group
```

Comment:
Tried enabling symlinks with `git` on Azure Windows builds, but am seeing this error [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=988057&view=logs&j=e5cdccbf-4751-5a24-7406-185c9d30d021&t=0c979b57-e97b-5e0a-363d-f964823a2ab0&l=532 )

```
error: could not lock config file /home/VssAdministrator/.gitconfig: No such file or directory
```

Reply:
Trying again with local (not global) symlinks

Reply:
Linking [this SO]( https://stackoverflow.com/q/55327408 ), which may have some clues